src/sql-parser/tests/testdata/explain

```diff
@@ -136,3 +136,26 @@ EXPLAIN TIMESTAMP FOR SELECT 1
 EXPLAIN TIMESTAMP FOR SELECT 1
 =>
 Explain(Old(ExplainStatementOld { stage: Timestamp, explainee: Query(Query { ctes: [], body: Select(Select { distinct: None, projection: [Expr { expr: Value(Number("1")), alias: None }], from: [], selection: None, group_by: [], having: None, options: [] }), order_by: [], limit: None, offset: None }), options: ExplainOptions { typed: false, timing: false } }))
+
+parse-statement
+EXPLAIN OPTIMIZED QUERY GRAPH TEXT PLAN WITH (monotonicity, types) FOR VIEW foo
```

Comment:
nit: in the original syntax, `QUERY GRAPH` is an alternative suffix to `PLAN`

```
EXPLAIN OPTIMIZED QUERY GRAPH
EXPLAIN OPTIMIZED PLAN
```

so we are not consistent with that syntax here. Maybe we can fix that?